contrib/packaging/deb/changelog_parser.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import sys
+from docutils.core import publish_doctree
+from datetime import datetime
+
+
+class ParseSection(object):
+
+    @property
+    def date(self):
+        return datetime.strptime(self._meta.get("date"), "%Y-%m-%d")
+
+    @property
+    def unixtime(self):
+        return int(self.date.strftime("%s"))
+
+    def __init__(self, section):
+        self._section = section
+
+    def parse(self):
+        if self._section.tagname != "section":
+            return False
+
+        children = self._section.children
+
+        self._parse_title(children[0])
+        self._parse_metadata(children[1])
+        self._parse_changelog(children[2:])
+        return self
+
+    def _parse_title(self, title):
+        self._title = title.astext()
+
+    def _parse_metadata(self, metadata):
+        self._meta = {}
+
+        for x in metadata:
+            self._meta[x.children[0].astext()] = x.children[1].astext()
+
+    def _parse_changelog(self, body):
+        self._changelog = ""
+        for section in body:
+            data = [x for x in section.astext().split("\n") if len(x) > 0]
+            self._changelog += "\n    ".join(data)
+
+    def export(self):
+        return """cilium ({1}) zesty; urgency=medium
+  * Changes:
+    commit: {4}
+    {0._changelog}
+
+ -- {2}  {3}
+        """.format(
+            self,
+            self._meta.get("version"),
+            self._meta.get("author"),
```

Comment:
Could we just hardcode this to `Authors of Cilium` or something similar?

Reply:
Need to be on this format: 
https://www.debian.org/doc/debian-policy/ch-source.html#s-dpkgchangelog

Maybe - Cilium Team <info@cilium.io>   date